src/progress-bar/__tests__/progress-bar.test.tsx

```diff
@@ -253,8 +261,10 @@ describe('Progress updates', () => {
     expect(wrapper.find(`.${liveRegionStyles.root}`)?.getElement().textContent).toBe(`${label}: 0%`);
     rerender(<ProgressBar label={label} value={2} />);
 
-    // 6 seconds passed, live region has a new value
-    jest.advanceTimersByTime(6000);
+    act(() => {
```

Comment:
Wrapped with an `act` because state updates are happening (https://github.com/cloudscape-design/components/blob/main/src/progress-bar/index.tsx#L55) when running the timers. This removes the related act warning when running the tests.